esrally/mechanic/launcher.py

```diff
@@ -184,8 +192,8 @@ def __init__(self, cfg, metrics_store, client_factory_class=client.EsClientFacto
         self.client_factory = client_factory_class
 
     def start(self, node_configurations=None):
-        hosts = self.cfg.opts("client", "hosts")
-        client_options = self.cfg.opts("client", "options")
+        hosts = self.cfg.opts("client", "hosts")()
+        client_options = self.cfg.opts("client", "options")()
```

Comment:
I think it would be better to use an explicit property name here instead of `()`?